language/ir-testsuite/tests/signer/double_signer.mvir

```diff
@@ -1,14 +1,14 @@
 main(s: &signer, s2: &signer) {
     return;
 }
-// check: INVALID_MAIN_FUNCTION_SIGNATURE
+// check: TYPE_MISMATCH
```

Comment:
I'm confused. This is the wrong error message. How did this get changed?

Reply:
I believe this is the correct error message: before two-signer txns were disallowed, however with this change two-signer transactions are allowed as long as they are a prefix. So this is no longer an invalid main signature, but a type error since two signers aren't passed through.

Reply:
that is a really large change to be doing right now no? How are we putting in more than one signer right now? I didn't see anywhere that did that

Reply:
It's not a terribly large change I would argue since this is only allowed for admin scripts (and passes both the sender and "as account" signers so has well-defined behavior). You can see this in the flow for the `AdminScript` I'll comment on this PR and call that part out.